src/mpm/staticmodbusmodel.py

```diff
@@ -3,12 +3,16 @@
 import attr
 import graham
 import marshmallow
+import typing
```

Comment:
I think it's fairly common convention in python to use `import typing as t/T` so make it shorter to add type annotations. No need to change here but I think we could adopt that convention